metadata-ingestion/transformers.md

```diff
@@ -210,6 +210,48 @@ transformers:
 ```
 
 Note that whatever owners you send via this will overwrite the owners present in the UI.
+### Add domain to dataset 
+let’s suppose we’d like to add a series of domain to dataset, in this case you can use `simple_add_dataset_domain` transformer.
+
+The config, which we’d append to our ingestion recipe YAML, would look like this:
+
+```yaml 
+transformers:
+  - type: "simple_add_dataset_domain"
+    config:
+      semantics: OVERWRITE
+      domain_urns:
```

Comment:
don't we support vanilla domains also? ones where you don't specify the urn?

Reply:
Yup we do support

Reply:
Added detail in readme

Reply:
that seems like something we should change ...

Reply:
the config should be called `domains:` and you should be able to provide urns or vanilla domain names.

Reply:
this would align with how the config for existing sources which have inbuilt domains capability works as well.

Reply:
Looks like the existing sources call it the `domain` field. e.g. https://datahubproject.io/docs/generated/ingestion/sources/bigquery

Reply:
@shirshanka I renamed the domain_urn to domains